dashboards/dashboard.json

```diff
@@ -1,21 +1,5 @@
 {
   "__inputs": [
-    {
-      "name": "DS_INFLUXDB_(AUTO PROVISIONED)",
-      "label": "InfluxDB (auto provisioned)",
-      "description": "",
-      "type": "datasource",
-      "pluginId": "influxdb",
-      "pluginName": "InfluxDB"
-    },
-    {
-      "name": "DS_SUN_AND MOON (AUTO PROVISIONED)",
-      "label": "Sun and Moon (auto provisioned)",
-      "description": "",
-      "type": "datasource",
-      "pluginId": "fetzerch-sunandmoon-datasource",
-      "pluginName": "Sun and Moon"
-    },
```

Comment:
Thinking through all the use cases that users may have...

There was a specific reason why we added the variables https://github.com/jasonacox/Powerwall-Dashboard/pull/512 - How will this work for existing users who didn't auto-provision?  All my test boxes are auto-provisioned at this point so I need to see if I can recreate what may be in the wild.